docs/nx-cloud/private/auth-saml.md

```diff
@@ -134,7 +134,7 @@ if you are interested.
     1. `SAML_CERT=<your-cert-string-from-above>`
     2. `SAML_ENTRY_POINT=<your-login-url-from-above>`
 
-# Helm config
+## Helm config
```

Comment:
Why is this file changed?

Reply:
because this creates multiple H1 tags on a page which causes playwright to fail.